tests/bcl-test/templates/common/TestRunner.xUnit/XUnitTestRunner.cs

```diff
@@ -1027,7 +1027,10 @@ public override void SkipTests (IEnumerable<string> tests)
 					} if (t.StartsWith ("KLASS64:", StringComparison.Ordinal) && IntPtr.Size == 8) {
 						var klass = t.Replace ("KLASS32:", "");
 						filters.Add (XUnitFilter.CreateClassFilter (klass, true));
-					}  else {
+					} if (t.StartsWith ("Platform32:", StringComparison.Ordinal) && IntPtr.Size == 4) {
```

Comment:
Either use `else if` or start the `if` statement on a new line, otherwise this looks confusing.

Reply:
Sorry, completely right.